src/tasks/copy-template-files.ts

```diff
@@ -213,7 +217,32 @@ const processTemplatedFiles = async (
         templateFileDescriptor.relativePath.split(templateTargetName)[0],
         templateTargetName,
       );
-      fs.writeFileSync(targetPath, output);
+
+      let finalOutput = output;
+
+      const containsNextjs = targetPath.includes("nextjs");
+      const containsDotfile = /\/\.[^/]+/.test(targetPath);
+      const hardhatOrFoundry = targetPath.split("packages")[1]?.split("/")[1];
```

Comment:
what about name it something like `extension` or `extensionFolder`?

Reply:
Ohh yeah my bad, just renamed it to `pacakgeName` at https://github.com/scaffold-eth/create-eth/pull/55/commits/9712cc3b8ab4ff978c88284618ab3bb3e69dac44